schema-source/ASHRAE229_extra.schema.yaml

```diff
@@ -241,15 +241,15 @@ RulesetModelDescription:
       AppG Used By TCDs: "unknown"
       AppG P_RMD Equals U_RMD: "unknown"
       AppG B_RMD Equals P_RMD: "unknown"
-      AppG B_RMDs Same: "unknown"
+      AppG B_RMDs Same: False
```

Comment:
I think this should probably be true. Can't think of when it would differ across the rotations.